### PR TITLE
Basic structure, team canvas and definition of done

### PR DESCRIPTION
**Pull request by DennisHartrampf**
@arosslau Please review and merge if appropriate ;)

### CODE REVIEW

File: Team Canvas.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
++ Information silos
++ Partly missing test systems
+
+## Activities
++ Build process knowledge
++ Implement monitoring together
++ Encourage internal and external knowledge transfer
++ Learn and use refactoring methods
++ Write testable code and test it
++ Team events
```
Comment (leberknecht) [inline Team Canvas.md]: Is this covered by singing songs in the dev room or is it refering to off-site events?

File: Team Canvas.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
++ Scrum
++ Admins on-site
+
+## Weaknesses
++ Second level support
++ Process knowledge
++ Bureaucratic processes (long approvals)
++ Refactoring methods (handling legacy code)
++ Still small team
++ Network connectivity (Proxy)
```
Comment (leberknecht) [inline Team Canvas.md]: can you be more specific here?

Comment (DennisHartrampf) [reply Team Canvas.md]: Thanks for your response! :)
You should be allowed to comment here, that's fine.
I'm not sure if I can be more specific here as this team canvas was developed in a team meeting and I just transcribed it here. I don't want to influence the meaning of this team effort. But maybe I can add some "editor's notes" to clarify some things.

File: Team Canvas.md (hunk 1 above)
Comment (DennisHartrampf) [reply Team Canvas.md]: This is up to your imagination ;)

**Pull request merged**